getting-started/jdbc/README.md

```diff
@@ -0,0 +1,92 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+ 
+   http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Getting Started with Apache Polaris, Relational JDBC, Postgres and Spark SQL
+
+This example requires `jq` to be installed on your machine.
```

Comment:
This might be covered by `install-dependencies.md`

cc @adnanhemani

Reply:
This is something we suggest in eclipse-link as well, hence here, happy to remove both.

Reply:
I think it's fine either way. Eric is right, but this being just mentioned here shouldn't be an issue.